packages/react-scripts/scripts/init.js

```diff
@@ -118,6 +118,10 @@ module.exports = function(appPath, appName, verbose, originalDirectory, template
     cdpath = appPath;
   }
 
+  // Change command to yarn instead of yarnpkg
+  if (useYarn)
+    command = 'yarn';
```

Comment:
Ready, if may i ask, why a separate variable? To avoid mutations of the command variable?